pystorm/bolt.py

```diff
@@ -63,21 +63,6 @@ def is_tick(tup):
         """ :returns: Whether or not the given Tuple is a tick Tuple """
         return tup.component == '__system' and tup.stream == '__tick'
 
-    def initialize(self, storm_conf, context):
```

Comment:
This was just a clean-up thing I noticed while I was working on this.  `initialize` is defined in `Component`, so this empty `Bolt` implementation is completely unnecessary.